pkg/distro/rhel9/edge.go

```diff
@@ -565,7 +586,7 @@ func edgeCommitPackageSet(t *imageType) rpmmd.PackageSet {
 		ps = ps.Append(aarch64EdgeCommitPackageSet(t))
 	}
 
-	if !common.VersionLessThan(t.arch.distro.osVersion, "9.2") || !common.VersionLessThan(t.arch.distro.osVersion, "9-stream") {
+	if !common.VersionLessThan(t.arch.distro.osVersion, "9.2") || !t.arch.distro.isRHEL() {
```

Comment:
This unrelated to the backported change, but not an issue... :thinking:

Reply:
yeah, not a problem at all :slightly_smiling_face:  this is harmless and even an additional fix :smiling_face_with_tear: